include/libtorrent/aux_/portmap.hpp

```diff
@@ -35,9 +35,8 @@ POSSIBILITY OF SUCH DAMAGE.
 
 #include "libtorrent/config.hpp"
 
-namespace libtorrent { namespace aux {
-	// TODO: move this for a better place and integrate it with
-	// portmap error alerts
+namespace libtorrent {
+
```

Comment:
if this will be in the public API, it should be moved to a place out of `aux_`

Reply:
yeah, I was thinking about maybe getting away with having ``alert_types.hpp`` and ``session.hpp`` be the public way of getting to these (a bit like how ``windows.h`` is the way to get to all other header files on windows basically). But it's probably better to break them out into a separate header.